src/editor/lib/SceneUtils.js

```diff
@@ -171,7 +189,9 @@ export async function saveScene(currentUser, doSaveAs) {
 export async function saveSceneWithScreenshot(currentUser, doSaveAs) {
   const currentSceneId = await saveScene(currentUser, doSaveAs);
   if (currentSceneId) {
-    makeScreenshot();
+    // wait a bit for models to be loaded, may not be enough...
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
I didn't have any issue locally, but I added that to wait a bit for models to be all loaded before taking the screenshot.